x-pack/plugins/cases/server/client/attachments/delete.ts

```diff
@@ -86,7 +86,7 @@ export async function deleteAll(
       concurrency: MAX_CONCURRENT_SEARCHES,
     });
 
-    await userActionService.bulkCreateAttachmentDeletion({
+    await userActionService.creator.bulkCreateAttachmentDeletion({
```

Comment:
All methods related to creating a user action or audit logging were moved to a new class. This class is accessible through the original user action service class via the `creator` public accessor method.